### ISSUE TITLE
Ignore files for reload

### ISSUE THREAD

**Issue by peterbarraud** [mattdesl/budo#219]
Is there a way to ignore files from reload? Something like watchify's: `--ignore-watch`?

**Comment by mattdesl** [comment]
Yes, good call. There is no simple way to achieve it across HTML, JS and CSS files, since the file watcher is split between watchify and budo's HTML/CSS watcher.

I can try to add something like this that works across the board.
